app/build.gradle

```diff
@@ -43,6 +43,16 @@ dependencies {
     compile 'com.android.support:cardview-v7:25.4.0'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
     compile 'me.dm7.barcodescanner:zxing:1.9.8'
+
+    androidTestCompile 'com.android.support:support-annotations:25.4.0'
+    androidTestCompile 'com.android.support.test:runner:0.5'
+    androidTestCompile 'com.android.support.test:rules:0.5'
+    // Optional -- Hamcrest library
+    //androidTestCompile 'org.hamcrest:hamcrest-library:1.3'
+    // Optional -- UI testing with Espresso
+    //androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
+    // Optional -- UI testing with UI Automator
+    //androidTestCompile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.2'
```

Comment:
why committing uncommented code?

Reply:
Do we have guidelines? The codebase is full of comments. I'm happy to take them off though

Reply:
I agree, Guidelines would be very helpful. In general I'm against uncommented code, imo if it is uncommented it is not needed.

Reply:
yes, please no commented-out code